gui/src/main/java/io/bisq/gui/app/BisqAppMain.java

```diff
@@ -68,6 +75,10 @@ public static void main(String[] args) throws Exception {
         // In order to work around a bug in JavaFX 8u25 and below, you must include the following code as the first line of your realMain method:
         Thread.currentThread().setContextClassLoader(BisqAppMain.class.getClassLoader());
 
+        if(options.has(AppOptionKeys.USE_DEV_PRIVILEGE_KEYS)) {
+            DevEnv.USE_DEV_PRIVILEGE_KEYS = true;
+        }
+
```

Comment:
This is not needed here.

But all the other places where the option keys are applied are missing.